bin/strap.sh

```diff
@@ -213,6 +217,23 @@ if ! [ -f "$HOME/.ssh/id_rsa" ]; then
   PUBLIC_KEY="$(cat $HOME/.ssh/id_rsa.pub)"
   POST_BODY="{\"title\":\"MacOSX Key - strap\",\"key\":\"$PUBLIC_KEY\"}"
   curl $Q -H "Content-Type: application/json" -H "Authorization: token $STRAP_GITHUB_TOKEN" -X POST -d "$POST_BODY" https://api.github.com/user/keys
+
+  log "checking for known_hosts file"
+  if ! [ -f "$HOME/.ssh/known_hosts"]; then
+    log "creating known_hosts file"
+    touch ~/.ssh/known_hosts
+    chmod 644 ~/.ssh/known_hosts
```

Comment:
I'm a fan of 600 for .ssh folder contents, but it's probably OK given all the other bugs with High Sierra...